tracer/build/_build/Build.GitHub.cs

```diff
@@ -1409,6 +1309,118 @@ private async Task<Milestone> GetMilestone(GitHubClient gitHubClient, string mil
         return allOpenMilestones.FirstOrDefault(x => x.Title == milestoneName);
     }
 
+    private async Task<int> GetConsolidatedPipelineBuildId(BuildHttpClient buildHttpClient, string targetBranch, string commitSha)
```

Comment:
Ultra nit - may not need a dedicated method as used only in `DownloadReleaseArtifacts` but feel free to ignore